RecoEcal/EgammaCoreTools/src/EcalClusterTools.cc

```diff
@@ -1307,8 +1307,11 @@ float EcalClusterTools::getNormedIX(const DetId& id)
     if( id.subdetId()==EcalEndcap ) ix = EEDetId(id).ix();
     if( id.subdetId()==EcalShashlik ) ix = EKDetId(id).ix();
     if( ix != 0) {
-      int iXNorm  = ix - 50;
-      if(iXNorm<=0) iXNorm--;
+      float iXNorm; 
+      //in shashlik ix and iy run from 1 to 210
+      if( id.subdetId()==EcalShashlik ) iXNorm  = ix - 105.5;
+      else iXNorm = ix - 50.5;
+      //      if(iXNorm<=0) iXNorm--;
```

Comment:
The cell id range from 100 to 210 is a bug fix. This is very good.
But changing iXNorm from an integer to a half-integer is just a mater of definition. There is no right or wrong, only better or less better. If you want to change it and say this is better please prove your point with proofs.